CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 * Field annotated with @Ignored should not have accessors generated by transformer (#2478).
 * RealmResults and RealmObjects can no longer accidentially be GC'ed if using `asObservable()`. Previously this caused the observable to stop emitting. (#2485).
+* Custom equals(), toString() and hashCode() work correctly now. 
```

Comment:
Perhaps "Custom equals(), toString() and hashCode() are no longer incorrectly overwritten by the proxy class (#xxxx)" instead?